src/_includes/partials/header.njk

```diff
@@ -1,40 +1,41 @@
 {% set otherLang = "fr" if locale == "en" else "en" %}
-<header>
-    <div  lang="{{ otherLang }}">
+<header class="header">
+    <div class="header__container" lang="{{ otherLang }}">
```

Comment:
I think we will need to move the lang attribute so it only surrounds the language toggle link. I don't think it will hurt anything where it currently is but just to be safe.

Reply:
Done!